example/SliderExample.js

```diff
@@ -1,240 +0,0 @@
-            })
-          }
-        />
-        <Text>
-          Completions: {this.state.slideCompletionCount} Value:{' '}
-          {this.state.slideCompletionValue}
-        </Text>
-      </View>
-    );
-  }
-}
-
-const styles = StyleSheet.create({
-  slider: {
-    width: 300,
-    opacity: 1,
-    height: 50,
-    marginTop: 10,
-  },
-  text: {
-    fontSize: 14,
-    textAlign: 'center',
-    fontWeight: '500',
-    margin: 0,
-  },
-});
-
-export const title = '<Slider>';
-export const displayName = 'SliderExample';
-export const description = 'Slider input for numeric values';
-export const examples = [
-  {
-    title: 'Default settings',
-    render(): Element<any> {
-      return <SliderExample />;
-    },
-  },
-  {
-    title: 'Initial value: 0.5',
-    render(): Element<any> {
-      return <SliderExample value={0.5} />;
-    },
-  },
-  {
-    title: 'minimumValue: -1, maximumValue: 2',
-    render(): Element<any> {
-      return <SliderExample minimumValue={-1} maximumValue={2} />;
-    },
-  },
-  {
-    title: 'step: 0.25, tap to seek on iOS',
-    render(): Element<any> {
-      return <SliderExample step={0.25} tapToSeek={true} />;
-    },
-  },
-  {
-    title: 'onSlidingStart',
-    render(): Element<any> {
-      return <SlidingStartExample />;
-    },
-  },
-  {
-    title: 'onSlidingComplete',
-    render(): Element<any> {
-      return <SlidingCompleteExample />;
-    },
-  },
-  {
-    title: 'Custom min/max track tint color',
-    render(): Element<any> {
-      return (
-        <SliderExample
-          minimumTrackTintColor={'#00FF00'}
-          maximumTrackTintColor={'red'}
-          value={0.5}
-        />
-      );
-    },
-  },
-  {
-    title: 'Custom thumb tint color',
-    render(): Element<any> {
-      return <SliderExample thumbTintColor={'magenta'} />;
-    },
-  },
-  {
-    title: 'Custom thumb image',
-    render(): Element<any> {
-      return <SliderExample thumbImage={require('./uie_thumb_big.png')} />;
-    },
-  },
-  {
-    title: 'Custom thumb (network image)',
-    platform: 'windows',
-    render(): Element<any> {
-      return (
-        <SliderExample
-          thumbImage={{
-            uri: 'https://img.icons8.com/windows/50/000000/bus.png',
-          }}
-        />
-      );
-    },
-  },
-  {
-    title: 'Custom track image',
-    platform: 'ios',
-    render(): Element<any> {
-      return <SliderExample trackImage={require('./slider.png')} />;
-    },
-  },
-  {
-    title: 'Custom min/max track image',
-    platform: 'ios',
-    render(): Element<any> {
-      return (
-        <SliderExample
-          minimumTrackImage={require('./slider-left.png')}
-          maximumTrackImage={require('./slider-right.png')}
-        />
-      );
-    },
-  },
-  {
-    title: 'Inverted slider direction',
-    render(): Element<any> {
-      return <SliderExample value={0.6} inverted />;
-    },
-  },
-  {
-    title: 'Vertical slider',
```

Comment:
this is not present in new example. I guess it's because windows platform. Do we have a plan to bring it back so we don't forget?

Reply:
Yes, @BartoszKlonowski temporarily disabled the build for windows for now, after the stable release for `react-native-windows@0.70` we will bring it back.

Reply:
Let's not worry about it for now, in my opinion we should completely revisit the way we present examples.
Currently we divide them by platforms, but since first implemented some features changed and, for example iOS does not contain the disabled prop even if it's available on iOS.
I didn't want to put all those cleanings and refactoring and stuff into single PR to avoid making mess.
So @okwasniewski  let's continue with keeping all examples even if they are currently unavailable, and let's focus on reorganizing it in further PRs.

Reply:
I've added it back, since it's not called on other platforms, and just to not lose it.